packages/wonder-blocks-button/docs.md

```diff
@@ -1,11 +1,6 @@
 **WARNING: Button is not finished, do not use!**
 
-Default Button:
-```js
-<Button>default button</Button>
-```
+A `Button` is a clickable button element consisting of a `ButtonBehavior` (TODO(yejia)) surrounding a `ButtonCore`. `ButtonBehavior` handles interactions and state changes. `ButtonBehavior` is a stateless component which displays the different states the `Button` can take.
 
-Caution Button:
-```js
-<Button color={"red"}>caution</Button>
-```
+### Usage
+TODO(yejia)
```

Comment:
Can you move button-core.md into this file? It would make for a great demonstration of all the possible states/sizes.